emcc.py

```diff
@@ -1861,6 +1861,10 @@ def include_and_export(name):
     if shared.Settings.USE_PTHREADS:
       newargs.append('-pthread')
 
+    # Any "pointers" passed to JS will now be i64's, in both modes.
+    if shared.Settings.MEMORY64:
```

Comment:
Just to be nice to the user:

```
If getattr(settings_key_changes, 'WASM_BIGINT') == '0':
   exit_wth_error('MEMORY64 is not compatible with WASM_BIGINT=0')
```

(We should probably make simpler way to express the above)